diff --git a/dist/schemas.d.ts b/dist/schemas.d.ts
index 11ad89722d50badc26d2792af7d9ee207d3c88e8..03ec07a9001412e076b9c746ebb41a8fea8df323 100644
--- a/dist/schemas.d.ts
+++ b/dist/schemas.d.ts
@@ -67,7 +67,7 @@ export interface BaseRun {
     /** Serialized state of the run for potential future use. */
     serialized?: object;
     /** Events like 'start', 'end' linked to the run. */
-    events?: KVMap[];
+    events?: KVMap[] | undefined;
     /** Inputs that were used to initiate the run. */
     inputs: KVMap;
     /** Outputs produced by the run, if any. */
